gio/src/file_info.rs

```diff
@@ -47,4 +47,28 @@ impl FileInfo {
             );
         }
     }
+
+    #[doc(alias = "g_file_info_get_attribute_stringv")]
+    #[doc(alias = "get_attribute_stringv")]
+    pub fn attribute_stringv(&self, attribute: &str) -> StrV {
+        unsafe {
+            FromGlibPtrContainer::from_glib_none(ffi::g_file_info_get_attribute_stringv(
```

Comment:
`FileInfo` is immutable, so this could also just be borrowed instead of creating a copy or not? (of course the same for everything else here)

I plan to add a configuration for making that possible with the new code generator, but as you say it will take some time.

Reply:
you are right, even the C api is `[transfer none]`

Reply:
`(transfer none)` in the C API is often a trap. Pointers that are only valid until *something* happens. It can only be borrowed directly if the underlying object is immutable (or it is a boxed type that is not refcounted).

Reply:
This is clearly such a case :) There's also a setter for this, and if you call the setter your pointer becomes invalid. As the info is refcounted and setting values uses interior mutability, this is unsound unfortunately and we always need to copy.

Reply:
ok, rolled back the change